lms/static/scripts/frontend_apps/components/YouTubePicker.tsx

```diff
@@ -53,8 +53,10 @@ export default function YouTubePicker({
   const onURLChange = (inputURL: string) => setCurrentURL(inputURL);
   const resetCurrentURL = () => setCurrentURL(undefined);
   const confirmSelection = () => {
-    if (videoId) {
-      onSelectURL(`youtube://${videoId}`);
+    if (currentURL) {
+      // YouTube video assignments use the raw YouTube URL for now.
+      // At some point we may change to a custom URL schema, like `youtube://video?url=YOUTUBE_URL
```

Comment:
Unless there is a likely hazard/confusion caused by using the original URL that readers need to be aware of, I think you can omit this comment as the code makes it obvious which URL is being used.